musicbot/constants.py

```diff
@@ -88,6 +111,13 @@
 # Maximum number of seconds to wait for HEAD request on media files.
 DEFAULT_MAX_INFO_REQUEST_TIMEOUT: int = 10
 
+# MusicBot can pre-download the track up-next, while the current plays.
+# Typically not an issue, but makes more requests over all to the same media.
+# If it is an issue for some, this constant can disable the pre-downloads.
+PRE_DOWNLOAD_NEXT_TRACK: bool = True
```

Comment:
This should probably be an option as opposed to having to edit code

Reply:
I'd agree, but wasn't sure if I should make it an option since it was default behavior to pre-download before.  Though since it is also extra requests, I thought "maybe it ought to be easier to disable for testing sake?"   Not really sure though.

Reply:
That default behavior was added by Bork, originally it never did way back when lol. Making it optionally is good, less request is muy Bueno.